packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -183,7 +183,8 @@ const Tooltip = React.forwardRef(function Tooltip(props, ref) {
     disableFocusListener = false,
     disableHoverListener = false,
     disableTouchListener = false,
-    enterDelay = 0,
+    enterDelay = 200,
```

Comment:
The spec seems fuzzy about this value. However, measuring from the videos of https://material.io/components/tooltips/#behavior, I come up with a value close to this.

It also matches a bit more experience on Gmail, Google Keep, and Google Calendar.